test/e2e/form-validation.test.js

```diff
@@ -186,9 +186,17 @@ describe('vl-form-validation', async () => {
     await form.submit();
     await assert.eventually.isTrue(input.hasError());
 
+    await input.setValue('2 0 00');
+    await form.submit();
+    await assert.eventually.isTrue(input.hasError());
```

Comment:
Het is hier misschien handig om, om zeker te zijn dat de validation goed werkt, te testen dat het afwisselend true, false, true, false is.

Reply:
ok, ik pas even de test aan

Reply:
volgorde aangepast